pkg/collector/corechecks/containers/containerd/utils.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+//go:build containerd
+// +build containerd
+
+package containerd
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/containers/generic"
+	"github.com/DataDog/datadog-agent/pkg/util/containers"
+	"github.com/DataDog/datadog-agent/pkg/workloadmeta"
+)
+
+func getProcessorFilter(legacyFilter *containers.Filter) generic.ContainerFilter {
+	// Reject all containers that are not run by Docker
```

Comment:
If I read it correctly, the filter below would reject containers not run by Containerd.